source/docs/03-metadata.xml

```diff
@@ -1382,156 +1354,308 @@
+                     <p></p>
+                  </div>
+                  <div xml:id="foliadesc" type="div4">
+                     <head>Description of folia</head>
+                     <p>While many other elements within <gi scheme="MEI">physDesc</gi> describe specific features of manuscripts and prints in prose, <gi scheme="MEI">foliaDesc</gi> is intended to be processable. It provides information about the binding of a manuscript or print and the layout of its pages. The most important elements used are:</p>
                      <p>
                         <specList>
-                           <specDesc key="patch" atts="attached.to"/>
+                           <specDesc key="foliaDesc"/>
+                           <specDesc key="folium"/>
+                           <specDesc key="bifolium"/>
                         </specList>
                      </p>
-                     <p>Depending on the parent, allowed values for <att>attached.to</att> are either "<hi rend="italic">recto</hi>" and "<hi rend="italic">verso</hi>" (in case of <gi scheme="MEI">folium</gi>) or "<hi rend="italic">outer.recto</hi>", "<hi rend="italic">inner.verso</hi>", "<hi rend="italic">inner.recto</hi>" and "<hi rend="italic">outer.verso</hi>" (in case of <gi scheme="MEI">bifolium</gi>).</p>
-                     <p>The exact position of the patch on the underlying surface may be specified using the optional <att>x</att> and <att>y</att> attributes, which are used to specify the distance from the upper left corner of the patch from the upper left corner of the surface it is attached to. At this point, it is not possible to specify rotation.</p>
-                     <p>The (optional) <att>attached.by</att> attribute specifies by which means the patch is attached. Suggested values are: "<hi rend="italic">glue</hi>" (patch is glued on surface beneath), "<hi rend="italic">thread</hi>" (patch is sewn on surface beneath), "<hi rend="italic">needle</hi>" (patch is pinned to the surface beneath), "<hi rend="italic">tape</hi>" (patch is taped on surface beneath using an adhesive strip) and "<hi rend="italic">staple</hi>" (patch is attached on surface beneath using a staple), but other values may be used as necessary.</p>
-                     <p>While the <gi scheme="MEI">patch</gi> element provides information about the attachment of a patch, the actual patch is encoded as a <gi scheme="MEI">folium</gi> or <gi scheme="MEI">bifolium</gi> child of <gi scheme="MEI">patch</gi>.</p>
+                     <p>The nesting of <gi scheme="MEI">bifolium</gi> and <gi scheme="MEI">folium</gi> elements reflects the nesting of paper sheets that make up the text block of the source. For instance, if a manuscript consists of two folded sheets of paper, with a single, unfolded sheet in the middle, this would be encoded with two nested <gi scheme="MEI">bifolium</gi> elements, where the inner one has an additional <gi scheme="MEI">folium</gi> element:</p>
```

Comment:
Folia are not always paper... vellum and parchment are very common materials in manuscripts.

Reply:
Suggestion to improve the description in follow-up PR's.

Reply:
I transfer work on this subsection to an issue for the Metadata IG.